### ISSUE TITLE
Amend the "Get set for our front end development projects" description to make clear to use Bootstrap 3.3.7

### ISSUE THREAD

**Issue by Marmiz** [freeCodeCamp/freeCodeCamp#13787]
#### Challenge Name
https://www.freecodecamp.com/challenges/get-set-for-our-front-end-development-projects

#### Issue Description

As of today Codpen default quick-add is Bootstrap.4 which has some differences from the 3.3.7 version that gets taught during the lessons.

I have noticed and increased number of Camper asking for help in chat asking why their code was not behaving as expected all due to them using version 4.

I propose to amend the description and provide the link to the version 3.3.7 and let the camper insert that.

It's also worth mentioning that in case we proceed with this change we have to change the `.gif` accordingly.

-----
[edit]
The text I'm referring to is at step 6/7:
> Click the gear in the upper left hand corner of the CSS box, then scroll down to "Quick add" and choose Bootstrap.

By doing so the camper will inevitably add the version 4.
We could change it with:
>Click the gear in the upper left hand corner of the CSS box, then in the text field add this link
`https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css`.

And perhaps leaving a note that using the dropdown will load the Alpha version, advising that may encounter some differences in syntax with the lessons's examples.

**Comment by systimotic** [comment]
@Marmiz Thanks for reporting! Good catch, we should take action to fix this. This will probably require a fix on `staging`, as well as a patch on `backup/master`

**Comment by Marmiz** [comment]
@systimotic I am happy to help with this :)

Also don't thank me, but the numbers of campers complaining in chat why the bootstrap code wasn't working 😉

**Comment by systimotic** [comment]
@Marmiz I do thank you, because you figured out what the problem was and reported it, helping not only the campers in the chat right now, but all campers who will see this challenge in the future 🙂

**Comment by Marmiz** [comment]
What a busy week!

Finally cut some time to work on this.
I've started by creating and uploading a new gif to imgur (like the previous one).
Here's the [link](http://imgur.com/oLYs8EA)

I had to create a new pen from an unregistered user so that my name won't show up :)
If this gif will be approved I will proceed with amending the text.
If not let me know an I'll be happy to modify accordingly.

![fcc - gif - imgur](https://cloud.githubusercontent.com/assets/16669241/23782591/1e6963e4-0588-11e7-933e-5acec6a13978.gif)

**Comment by systimotic** [comment]
Hi @Marmiz! As you can tell from the late response, I'm no stranger to busy weeks 😅

Opening a PR anyway was a good move, that way other people might pick this up as well 👍 

I'm a bit worried about the quality of the new gif.
https://i.imgur.com/G9KFQDL.gif old
https://i.imgur.com/oLYs8EA.gif  new
I think the new gif could be sped up a bit as well. As an onlooker, looking at the typing is really slow, especially because the typing doesn't really matter, I only need to see what it does.

**Comment by Marmiz** [comment]
Hello @systimotic, unfortunately the poor quality is due to imgur compressing :(
I had recorded it in on a retina display in high res.

The I used the imgur built-in tool to create a gif out of the video; I believe that that's where the compressing came from.

Plus the general slowness is due to github/our website I believe, the gif is exactly 13 seconds long :)

Perhaps we should use another tool to create the gif to avoid too much quality compression?
Anything to recommend?
